docs/testing.md

```diff
@@ -0,0 +1,26 @@
+# Testing
+This project has different tests. All tests run in GitHub CI after creating a pull request. Therefore, pull requests can only be merged on successful tests.
+
+## Unit testing with Jest
+We use [Jest](https://jestjs.io/) for our unit tests. You can run the tests with the following:
+
+  npm run test
```

Comment:
Or add this later - we currently don't have unit test? But I could add a section about integration tests.

Reply:
I changed 'Unit tests' to 'Integration tests'. You can change/update this section if you want.